plugins/woocommerce/client/blocks/assets/js/atomic/blocks/product-elements/image/block.json

```
@@ -5,7 +5,7 @@
 	"category": "woocommerce-product-elements",
 	"attributes": {
 		"showProductLink": { "type": "boolean", "default": true },
-		"showSaleBadge": { "type": "boolean", "default": true },
+		"showSaleBadge": { "type": "boolean", "default": false },
```

I assume it's leftover, we should not change it for backwards compatibility reasons

```suggestion
		"showSaleBadge": { "type": "boolean", "default": true },
```